src/librustc_resolve/lib.rs

```diff
@@ -730,6 +741,14 @@ enum UseLexicalScopeFlag {
     UseLexicalScope,
 }
 
+#[allow(dead_code)]
+enum LookupTarget {
+    // Lookup only in the current crate
+    CurrentCrate,
```

Comment:
Yeah, I would remove it -- I can't think of a reason to restrict lookup to crate local items.